src/components/DropDownMenu.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Checked from '../images/iconfinder_done-01_186405.svg'
+import CheckedBlack from '../images/iconfinder_icon-checkmark_black.svg'
+export const DropDownMenu = ({ isOpen, setFormat, format, formatOptions, settings: { dark } }) => {
+  const imgToShow = dark ? Checked : CheckedBlack
+  return (
+    <div>
+      {
+        isOpen && <div style={dark ? {
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'space-evenly',
+          border: '1px solid white',
+          width: '108px',
+          height: '65px'
+        } : {
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'space-evenly',
+          border: '1px solid black',
+          width: '108px',
+          height: '65px'
+        }}>
```

Comment:
Notice that these are identical except for the border. You should use the ternary just on the border property, not the whole object. This will eliminate the redundancy and make it clearer what is different.